src/components/Navbar/index.tsx

```diff
@@ -14,7 +15,7 @@ const Navbar = (): ReactElement => {
   };
 
   return (
-    <Header className={styles.navbar}>
+    <Header className={styles.navbar} style={{ paddingInline: '0' }}>
```

Comment:
Is this not the same as what you're importing from the AntD overrides?

Reply:
It is... turns out the overrides didn't work. Not specific enough, so I put this in for now until I can work out what specificity is needed, if it is indeed possible